optimade/validator/validator.py

```diff
@@ -564,11 +853,9 @@ def test_page_limit(self, response, check_next_link: int = 5) -> (bool, str):
             check_next_link (int): maximum recursion depth for following
```

Comment:
Also, remove the two lines above, putting this together with the list of `Parameters`.
```suggestion
            check_next_link: Maximum recursion depth for following
```